tensorboard/webapp/notification_center/_redux/notification_center_types.ts

```diff
@@ -29,9 +29,11 @@ export interface Notification {
 }
 
 export const NOTIFICATION_FEATURE_KEY = 'notification';
+export const NOTIFICATION_LAST_READ_TIME_KEY = 'lastReadNotificaitonTimestamp';
 
 export interface NotificationState {
   notifications: Notification[];
+  lastReadTimestamp: number;
```

Comment:
I think this should be nullable to indicate that it has not been loaded yet.

For instance, if we were persist this information in the backend, we want to make this state fetchable from the backend and we have to account for the flight time and should be able to tell difference between 'not fetched yet' and UTC time = 0.

Reply:
added null type. then return `-1` in `getLastReadTime` selector. Please let me know any suggestions.